apps/files_sharing/src/views/SharingDetailsTab.vue

```diff
@@ -1056,9 +1056,15 @@ export default {
 						flex-direction: column;
 					}
 				}
-
-				.subline {
-					display: block;
+				
+				/* Target component based style in NcCheckboxRadioSwitch slot content*/
+				::v-deep span.checkbox-content__text.checkbox-radio-switch__text {
```

Comment:
v-deep is deprecated, use deep instead

(vue3 docs but also applies to vue 2.7): https://vuejs.org/api/sfc-css-features.html#deep-selectors

```suggestion
				:deep(span.checkbox-content__text.checkbox-radio-switch__text) {
```